test/e2e/network/networking.go

```diff
@@ -81,24 +81,13 @@ var _ = common.SIGDescribe("Networking", func() {
 	var svcname = "nettest"
 	f := framework.NewDefaultFramework(svcname)
 
-	ginkgo.BeforeEach(func() {
-		// Assert basic external connectivity.
-		// Since this is not really a test of kubernetes in any way, we
-		// leave it as a pre-test assertion, rather than a Ginko test.
-		ginkgo.By("Executing a successful http request from the external internet")
-		resp, err := http.Get("http://google.com")
-		if err != nil {
-			framework.Failf("Unable to connect/talk to the internet: %v", err)
-		}
-		if resp.StatusCode != http.StatusOK {
-			framework.Failf("Unexpected error code, expected 200, got, %v (%v)", resp.StatusCode, resp)
-		}
-	})
-
 	ginkgo.It("should provide Internet connection for containers [Feature:Networking-IPv4]", func() {
 		ginkgo.By("Running container which tries to connect to 8.8.8.8")
```

Comment:
any objections to delete this test given the new one below?

Reply:
I'll keep both, one test direct IP connectivity, the other tests that you also resolve www.google.com

Reply:
also, if we are pedantic, google.com is not really ipv4 or ipv6, it will depend on the resolver

Reply:
fair point, I'm just gonna put it under a new ginkgo.It

Reply:
you have to add it a Feature tag or it will be running by default, or do you want to filter based on the description?

Reply:
I updated the feature filters to `[Feature:Internet-*]`. I don't think the existing feature tags `Networking-*` are useful since they are too generic and they don't seem to used much outside this package.